sdk/ai/ai-projects/package.json

```
@@ -0,0 +1,159 @@
+{
+  "name": "@azure/ai-projects",
+  "version": "1.0.0-beta.1",
+  "description": "A generated SDK for ProjectsClient.",
+  "engines": {
+    "node": ">=18.0.0"
+  },
+  "sideEffects": false,
+  "autoPublish": false,
+  "tshy": {
+    "exports": {
+      "./package.json": "./package.json",
+      ".": "./src/index.ts"
+    },
+    "dialects": [
+      "esm",
+      "commonjs"
+    ],
+    "esmDialects": [
+      "browser",
+      "react-native"
+    ],
+    "selfLink": false
+  },
+  "type": "module",
+  "keywords": [
+    "node",
+    "azure",
+    "cloud",
+    "typescript",
+    "browser",
+    "isomorphic"
+  ],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "files": [
+    "dist",
+    "README.md",
+    "LICENSE",
+    "review/*",
+    "CHANGELOG.md"
+  ],
+  "sdk-type": "client",
+  "repository": "github:Azure/azure-sdk-for-js",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "prettier": "@azure/eslint-plugin-azure-sdk/prettier.json",
+  "//metadata": {
+    "constantPaths": [
+      {
+        "path": "src/generated/src/projectsClient.ts",
+        "prefix": "userAgentInfo"
+      },
+      {
+        "path": "src/constants.ts",
+        "prefix": "SDK_VERSION"
+      }
+    ]
+  },
+  "dependencies": {
+    "@azure-rest/core-client": "^2.1.0",
+    "@azure/abort-controller": "^2.1.2",
+    "@azure/core-auth": "^1.6.0",
+    "@azure/core-rest-pipeline": "^1.5.0",
+    "@azure/core-util": "^1.9.0",
+    "@azure/logger": "^1.1.4",
+    "@azure/core-lro": "^2.0.0",
+    "tslib": "^2.6.2",
+    "@azure/core-paging": "^1.5.0",
+    "@azure/core-sse": "^2.1.3",
+    "@azure/core-tracing": "^1.2.0"
+  },
+  "devDependencies": {
+    "@azure/dev-tool": "^1.0.0",
+    "@azure/eslint-plugin-azure-sdk": "^3.0.0",
+    "@azure/identity": "^4.3.0",
+    "@azure/opentelemetry-instrumentation-azure-sdk": "^1.0.0-beta.7",
+    "@azure/monitor-opentelemetry-exporter": "^1.0.0-beta.27",
+    "@azure-tools/test-credential": "^2.0.0",
+    "@azure-tools/test-recorder": "^4.1.0",
+    "@azure-tools/test-utils-vitest": "^1.0.0",
+    "@microsoft/api-extractor": "^7.40.3",
+    "@opentelemetry/api": "^1.9.0",
+    "@opentelemetry/instrumentation": "0.53.0",
+    "@opentelemetry/sdk-trace-node": "^1.9.0",
+    "@vitest/browser": "^2.0.5",
+    "@vitest/coverage-istanbul": "^2.0.5",
+    "@types/node": "^18.0.0",
+    "dotenv": "^16.0.0",
+    "eslint": "^8.55.0",
+    "prettier": "^3.2.5",
+    "rimraf": "^5.0.5",
+    "mkdirp": "^3.0.1",
+    "playwright": "^1.41.2",
+    "typescript": "~5.5.3",
+    "tshy": "^1.11.1",
+    "vitest": "^2.0.5"
+  },
+  "scripts": {
+    "clean": "rimraf --glob dist dist-browser dist-esm test-dist temp types *.tgz *.log",
```

for some dev dependencies we migrated to use centralized version
```suggestion
    "clean": "dev-tool run vendored rimraf --glob dist dist-browser dist-esm test-dist temp types *.tgz *.log",
```